index.html

```diff
@@ -31,13 +32,213 @@ <h2 id="job-title">I am a Software Developer</h2>
+                        </div>
+                        <div>
+                            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <ul class="project-1-buttons-ul">
+                            <a href="#"><button class="project-1-buttons">html</button></a>
+                            <a href="#"><button class="project-1-buttons" >bootstrap</button></a>
+                            <a href="#"><button class="project-1-buttons">Ruby</button></a>
+                        </ul>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button class="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                <div class="project-1">
+                    <img src="images/Img Placeholder-2.png" alt="printed-magazine">
+                    <div class="floating">
+                        <div>
+                            <h3 class="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <ul class="project-1-buttons-ul">
+                            <a href="#"><button class="project-1-buttons">html</button></a>
+                            <a href="#"><button class="project-1-buttons" >bootstrap</button></a>
+                            <a href="#"><button class="project-1-buttons">Ruby</button></a>
+                        </ul>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button class="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                <div class="project-1">
+                    <img src="images/Img Placeholder-2.png" alt="printed-magazine">
+                    <div class="floating">
+                        <div>
+                            <h3 class="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <ul class="project-1-buttons-ul">
+                            <a href="#"><button class="project-1-buttons">html</button></a>
+                            <a href="#"><button class="project-1-buttons" >bootstrap</button></a>
+                            <a href="#"><button class="project-1-buttons">Ruby</button></a>
+                        </ul>
```

Comment:
- [ ] The same also here; please use the proper unorder list syntax, `ul and li` go the same as a parent and child structure like the example below :+1: .

Example
```
  <ul>
      <li><a href="#">html</a></li>
      <li><a href="#">css</a></li>
      <li><a href="#">bootstrap</a></li>
  </ul>
```